src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

```diff
@@ -27,6 +27,85 @@ With Azion, you can customize the time the cache remains in the edge through *ti
 
 ---
 
+## Cache keys
+
+A cache key is an index entry for an object in Azion's edge nodes. When a request is made to an edge application for the first time, the edge forwards the request to the origin server and generates a cache for each object received at the edge. When the next request for the same resource is made to the edge application, if the resource requested matches one already found through the cache key, the request won't be forwarded to the origin.
+
+You can obtain the cache key of an application's resource through a `HEAD` request for the content URI and add the Azion debug header `Pragma: azion-debug-cache`:
+
+```bash
+curl --location --head 'https://yourdomain.com/resource-path/image.jpeg' \
```

Comment:
--head is not mandatory.

Reply:
Simplified the command and explanation 😉